UserAgent/Home View/BookmarksView.swift

```diff
@@ -0,0 +1,240 @@
+//
+// Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+
+import Shared
+import Storage
+import UIKit
+
+// MARK: - Placeholder strings for Bug 1232810.
+let deleteWarningTitle = NSLocalizedString("This folder isn’t empty.", tableName: "BookmarkPanelDeleteConfirm", comment: "Title of the confirmation alert when the user tries to delete a folder that still contains bookmarks and/or folders.")
```

Comment:
Do we need to move all localizations in String extension under Shared folder? Let's discuss it during today's meeting.